docs/content/posts/2023-08-03-externalized-connectors.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Announcing three new Apache Flink connectors, the new connector versioning strategy and externalization"
+date: "2023-08-03T12:00:00Z"
+authors:
+- elphastori: 
+  name: "Elphas Toringepi"
+  twitter: "elphastori"
+aliases:
+- /news/2023/08/03/externalized-connectors.html
+---
+
+## New connectors
+
+We're excited to announce that Apache Flink now supports three new connectors: [Amazon DynamoDB](https://aws.amazon.com/dynamodb), [MongoDB](https://www.mongodb.com/) and [OpenSearch](https://opensearch.org/)! The connectors are available for both the [DataStream](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/overview/) and [Table/SQL](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/overview/) APIs.  
+
+- **[Amazon DynamoDB](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/dynamodb/)** - This connector includes a sink that provides at-least-once delivery guarantees.
+- **[MongoDB connector](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/mongodb/)** - This connector includes a source and sink that provide at-least-once guarantees.
+- **[OpenSearch sink](https://github.com/apache/flink-connector-opensearch/blob/main/docs/content/docs/connectors/datastream/opensearch.md)** - This connector includes a sink that provides at-least-once guarantees
+
+|Connector|Date Released|Supported Flink Versions|
+|---|---|---|
+|Amazon DynamoDB sink|12/2/2022|1.15+|
+|MongoDB connector|3/31/2023|1.16+|
+|OpenSearch sink|12/21/2022|1.16+|
```

Comment:
nit: Are these date formats used elsewhere? Would prefer yyyy-mm-dd personally

Reply:
I prefer ISO as well. I've changed the format.